plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/storage/KVMStorageProcessor.java

```diff
@@ -272,8 +272,17 @@ public Answer copyTemplateToPrimaryStorage(final CopyCommand cmd) {
 
                 String path = derivePath(primaryStore, destData, details);
 
-                if (!storagePoolMgr.connectPhysicalDisk(primaryStore.getPoolType(), primaryStore.getUuid(), path, details)) {
-                    s_logger.warn("Failed to connect physical disk at path: " + path + ", in storage pool id: " + primaryStore.getUuid());
+                if (path == null) {
+                    path = destTempl.getUuid();
+                }
```

Comment:
The problem is now rather this.
Linstor ignored path == `null` requests before and kept going, but as this is now setting a UUID that doesn't exist yet, so the connect fails and the Linstor adapter will throw an exception not the code below.

So if we want to keep it ignoring it, we will have to keep the `null` path here.